plutus-playground-server/src/Playground/Interpreter.hs

```diff
@@ -213,7 +213,10 @@ runghcOpts =
     , "-fno-ignore-interface-pragmas"
     , "-fobject-code"
     -- FIXME: stupid GHC bug still
-    , "-package plutus-tx"
+    -- if you want to run in nix include plutus-tx but not plutus-wallet-api
+    -- if you want to run in stack include plutus-wallet-api but not plutus-tx
```

Comment:
I wish it were that simple... tbh I think it's random or based on some sorting thing that changes based on the hash or something.

Reply:
blurgh. This has always managed to work for me but I've always used the same computer or maybe I've just been 'lucky'

Reply:
Can you remove the comment since I think it's not necessarily true? Thanks.